content/index.html.haml

```diff
@@ -137,6 +136,8 @@ homepage: true
             helping drive builds, tests and deployments across multiple
             platforms faster.
 
+= partial('video.html.haml')
```

Comment:
purely an idea, i don't know if haml can handle it.
but could it be called youtube.html.haml, and you pass in the youtube id, and then you wouldn't need the rand stuff then you could just use the video id as the unique identifier?

Reply:
The unique id is used during page development to allow me to refresh page and get a new identifier each time.  I copied the pattern from the projectjumbotron.html.haml fragment and found that it worked well during page development.  Without that, I believe @rtyler and others had found that page development sometimes gave misleading results when the CSS was not replaced on a page reload.

In this case, I barely know enough haml and css to copy existing code (yes, I used "cargo cult programming" for this change).  I may be completely wrong and am happy to have anyone with more CSS and HTML skill guide me.